tests/avocado/README.rst

```diff
@@ -0,0 +1,40 @@
+This directory is hosting functional tests written using Avocado Testing
+Framework. To install Avocado, follow the instructions from this link::
+
+    http://avocado-framework.readthedocs.io/en/latest/GetStartedGuide.html#installing-avocado
+
+Tests here are written keeping the minimum amount of dependencies. So
+far, you only need the Avocado core package (`python-avocado`) to run
+the tests. Extra dependencies introduced in the future should be
+documented here in this README.rst file.
+
+In this directory, a ``base`` module is provided in order to tool test
```

Comment:
From a non-native speaker: I'm not sure "tool" can be used as a verb... maybe it can, but it sounds a bit "different" to me.